src/Variables/DefaultVariables.jl

```diff
@@ -5,7 +5,8 @@ export Circular, Circle
 
 
 Base.convert(::Type{<:Tuple}, ::InstanceType{Manifolds.Euclidean{Tuple{N}, ℝ}} ) where N = tuple([:Euclid for i in 1:N]...)
-Base.convert(::Type{<:Tuple}, ::InstanceType{Manifolds.Circle{ℝ}})  = (:Circular,)
+Base.convert(::Type{<:Tuple}, ::InstanceType{Manifolds.Circle{ℝ}})  = error("#FIXME")#(:Circular,)
+Base.convert(::Type{<:Tuple}, ::InstanceType{Manifolds.RealCircleGroup})  = (:Circular,)
```

Comment:
yeah, I think we should just ditch the old tuples from IIF v0.25 entirely.  Doesn't block this PR though, must just remember to do before release.